docs/source/guides/specifying_primitive_options.rst

```diff
@@ -0,0 +1,75 @@
+Specifying Primitive Options
+============================
+
+By default, DFS will apply primitives across all entities and columns. This behavior can be altered through a few different
+parameters. Entities and variables can be optionally ignored or included for an entire DFS run or on a per-primitive basis,
+enabling greater control over features and less run time overhead.
+
+
+Specifying Options for an Entire Run
+************************************
+The ``ignore_entities`` and ``ignore_variables`` parameters of DFS control entities and variables (columns) that should be
+ignored for all primitives::
+
+    # ignore the 'sessions' entity and the 'session_id' variable in 'transactions'
+    feature_matrix, features_list = ft.dfs(entityset=es,
+                                           ignore_entities=['sessions'],
+                                           ignore_variables={'transactions': ['session_id']})
+
+DFS will then ignore the ``sessions`` entity and the variable ``session_id`` in ``transactions`` when creating features.
+However, both of these options can be overridden by individual primitive options in the ``primitive_options`` parameter.
+
+Specifying for Individual Primitives
+************************************
+Options for individual primitives or groups of primitives are set by the ``primitive_options`` parameter of DFS. This parameter
+maps any desired options to specific primitives. In the case of conflicting options, options set at this level will override
+options set at the entire DFS run level, and the include options will always take priority over any ignore options.
+
+Specifying Entities for Individual Primitives
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
is it possible to to turn this into runnable examples like we? that way when we build the documentation, we would get notified if the examples ever break. it also makes it easier for a reader to see what is going on without running the code themselves